packages/liveblocks-client/src/index.ts

```diff
@@ -24,6 +24,7 @@ export type {
   CommentBodyText,
   CommentBodyTextElementArgs,
   CommentData,
+  CommentReaction,
```

Comment:
This was only exported from `@liveblocks/core`, and not yet from `@liveblocks/client`.